### ISSUE TITLE
Mirror/replicate from a starting offset?

### ISSUE THREAD

**Issue by daluu** [uber/uReplicator#156]
A thought just came to mind, for replication/mirroring, can uReplicator start the process against source topics at a given offset (or say from earliest offset)? Or is it always going to be from latest offset?

**Comment by xhl1988** [comment]
Yes, just specify `auto.offset.reset=smallest` in [consumer.properties](https://github.com/uber/uReplicator/blob/master/config/consumer.properties)

**Comment by cricket007** [comment]
And what about a specific offset? Or does one need to play with manual consumer group offset assignment?

**Comment by xhl1988** [comment]
Currently we don't support that in the config. But you can manually override the offset in zookeeper.

**Comment by cricket007** [comment]
Zookeeper?? Let's assume we're on a modern Kafka version 😊 

The `__consumer_offsets` topic makes more sense. And `kafka-consumer-groups --reset-offsets` could be used

**Comment by xhl1988** [comment]
uReplicator uses zookeeper for now (we might migrate to offset topic later). But the idea is the same that you have to manually override the offset either in zk or __consumer_offsets.
